### ISSUE TITLE
Template conversion issues/bugs

### ISSUE THREAD

**Issue by euantorano** [mybb/mybb#3294]
Originally posted here: https://community.mybb.com/thread-215373-post-1307516.html#pid1307516

- [ ] announcement: Not Display in Forums
- [ ] announcement_edit: Work Perfect in Admin CP but in Mod CP not Display
- [ ] announcement_quickdelete:  Work Perfect in Admin CP but in Mod CP not Display
- [ ] almost everything works well with the calendars except the moderation part
- [ ] forumdisplay_thread_rating  Not Display.
- [ ] /stats.php lost style
- [ ] Much Memory Used in Statistics Report on mybb 1.9 

MyBB 1.9: 

```
Generated in 214 ms with 17 queries. Memory Usage: 6 MB
```

MyBB 1.8:

```
Generated in 862 ms with 17 queries. Memory Usage: 2 MB
```
- [ ] that happened when try to loggin in MyBB 1.9

> You have entered an invalid username/password combination. <br /><br />If you have forgotten your password please <a href="member.php?action=lostpw">retrieve a new one</a>.

- [ ] Buddy request received, when try to respon via pm display:

> One or more of the recipients you specified are not registered usernames. Invalid recipients: Array

**Comment by Azareal** [comment]
If the memory is traded for speed, then I'm all for that. But, the general rule is that higher memory usage reduces speed (there are exceptions, if you're using it in a more intelligent way ;)), as it results in more data being copied around, etc.

It could just be noise creating the speed difference there though, maybe try it with APC enabled and a SSD, that tends to make things whiz by really fast.
For the most part, eval drags the speed down, as it does parsing on the spot rather than reusing the work like opcache does, plus it's insecure.

**Comment by euantorano** [comment]
Yes, as I said in the thread [Twig is in debug mode](https://github.com/mybb/mybb/blob/develop/1.9/inc/src/bootstrap.php#L70). I'm not sure how much difference that makes int erms of memory usage but it is definitely worth noting before doing any benchmarking.

Other additions such as [the container](https://github.com/mybb/mybb/blob/develop/1.9/inc/src/bootstrap.php#L15) (which stores registered factories and instances in memory) will also bump memory usage up slightly, along with the event dispatcher.

**Comment by rajat315315** [comment]
> forumdisplay_thread_rating Not Display.

Aren't we deprecating thread ratings in 1.9?

**Comment by arcalys** [comment]
- Announcements display in forums ✅ 
- Announcements edition in ModCP ✅ 
- Announcements quick delete in ModCP ✅ 
- Calendar moderation ✅ 
- Thread rating deprecation has been merged ✅ 
- Forum statistics style ✅ 
- Memory "issue" seems to be gone, but if not should probably tackled separately.
- The login issue has been fixed AFAIK.
- Buddy list issue is fixed ✅

**Comment by arcalys** [comment]
@Ben-MyBB 
I think this can be closed.
